externs/firebaseui-externs.js

```diff
@@ -231,7 +231,8 @@ firebaseui.auth.Config.prototype.signInFlow;
  * Determines the list of IdPs for handling federated sign-in, phone number as
  * well as password account sign-up.
  *
- * @type {!Array<!firebaseui.auth.SignInOption>|undefined}
+ * @type {!Array<!firebaseui.auth.SignInOption|string>
```

Comment:
Could you please put them on the same line?

Reply:
I did!